### ISSUE TITLE
Tracking renku-core-service tasks

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku-ui#966]
## Motivation

Tasks on the renku service are asynchronous and may need time to complete. The UI should inform the user of these situations and provide an overview of the tasks that are running.

### Top-level Task Pagea

Add a Task page to the top-level navigation. This page should summarize any tasks that the renku service is running for the user.

<img width="684" alt="image" src="https://user-images.githubusercontent.com/1196411/85726471-67ecbe80-b6f6-11ea-9bfe-355f99d25809.png">

### Project/Overview/Tasks

Add a tab "Tasks" to the Project/Overview page. The tab should be shown if tasks are running and should be hidden otherwise.

<img width="522" alt="image" src="https://user-images.githubusercontent.com/1196411/85726659-979bc680-b6f6-11ea-9fa2-4250cc429192.png">

### Project/Datasets

If a task is running, it must complete before any operations can be done with the repo. On the Project/Datasets page, show a banner that a task is running with a link to the Project/Overview/Tasks page. "New Dataset" and "Add File" should be disabled with a tooltip that these operations cannot be done until tasks complete running.

<img width="756" alt="image" src="https://user-images.githubusercontent.com/1196411/85726893-d5005400-b6f6-11ea-8b1c-43860dc47e40.png">

**Comment by rokroskar** [comment]
Should the KG generation be added to the same list?

**Comment by ciyer** [comment]
> Should the KG generation be added to the same list?

This is lower priority because of #977.

**Comment by lorenzo-cavazzi** [comment]
Renku-core isn't relevant anymore in [Renku 2.0](https://blog.renkulab.io/renku-2) and the concept of Task/Workflow will be revised.
I'm closing this issue. Feel free to open a new one if you think there are still reasons to consider a similar functionality.
